react-web/src/components/supportSchedules/SupportSchedules.jsx

```diff
@@ -2,27 +2,40 @@ import React, { useState, useEffect } from 'react';
 import { Table, Container, Row, Col, Form, Button, Pagination } from 'react-bootstrap';
 import { FaEdit, FaEye, FaTrashAlt } from 'react-icons/fa';
 import { Tooltip } from "@mui/material";
-import { format } from 'date-fns';
 import useCrud from '../../hooks/useCrud';
+import { format } from 'date-fns';
 import './SupportSchedulesStyles.css';
 import SupportScheduleModal from './SupportScheduleModal';
 import { ReactComponent as AddIcon } from '../../assets/svgIcons/add.svg';
 import OpenModalButton from '../common/OpenModalButton';
-
+import moment from 'moment';
 
 const SupportSchedules = () => {
     const { getAll } = useCrud();
     const [supportSchedules, setSupportSchedules] = useState([]);
     const [selectedSupportSchedule, setSelectedSupportSchedule] = useState(null);
     const [modalType, setModalType] = useState(null);
-    const userUrl = "/users";
+    const supportTeamSchedulesUrl = "/support-schedules";
 
     const [page, setPage] = useState(1);
     const [rowsPerPage, setRowsPerPage] = useState(5);
     const [searchQuery, setSearchQuery] = useState('');
     const [sortField, setSortField] = useState('');
     const [sortOrder, setSortOrder] = useState('asc');
 
+    // Bangladesh time zone
+    const timeZone = 'Asia/Dhaka';
+    // const inputTimeString = '20:00:00';
+
+    // // Parse the input time string into a Date object (assuming UTC)
+    // const inputTime = new Date(inputTimeString);
+
+    // // Convert the UTC time to zoned time in Bangladesh
+    // const zonedTime = zonedTimeToUtc(inputTime, timeZone);
+
+    // // Format the zoned time for display
+    // const formattedTime = format(zonedTime, 'h:mm:ss a z');
```

Comment:
Is setting timezone necessary? If necessary then take this from user locale timezone.